support/tools/go.mod

```diff
@@ -1,8 +1,147 @@
 module github.com/filecoin-project/specs-actors/v6/support/tools
 
-go 1.16
+go 1.19
```

Comment:
The current go version is 1.17. The next go 1.18. How are you running go 1.19?????

Reply:
uhhh the jetlag caused me to time-travel forward to like 2022 (whoops! fixed it though, good catch)